src/main/java/io/github/genomicdatainfrastructure/daam/gateways/RemsApplicationMapper.java

```diff
@@ -282,15 +283,17 @@ private List<RetrievedApplicationLicense> toLicences(Application application) {
     private RetrievedApplicationLicense toLicense(V2License license) {
         var potentialLicense = ofNullable(license);
 
-        return new RetrievedApplicationLicense(
-                potentialLicense.map(this::toLicenseType)
-                        .orElse(null),
-                potentialLicense.map(l -> toLabelObject(l.getLicenseTitle()))
-                        .orElse(null),
-                potentialLicense.map(V2License::getLicenseEnabled)
-                        .orElse(null),
-                potentialLicense.map(V2License::getLicenseArchived)
-                        .orElse(null));
+        return RetrievedApplicationLicense.builder()
+                .type(potentialLicense.map(this::toLicenseType).orElse(null))
+                .title(potentialLicense.map(l -> toLabelObject(l.getLicenseTitle())).orElse(null))
```

Comment:
important: please ensure `toLabelObject` and `convertMapToString` feel odd, it looks there is a mapping issue. Can we discuss to understand what is happening here?